worlds/inscryption/Rules.py

```diff
@@ -0,0 +1,203 @@
+            "Act 3 - Nano Armor Generator": self.has_gems_and_battery,  # Costs money, so can need multiple battles.
+            "Act 3 - Shop Holo Pelt": self.has_gems_and_battery,  # Costs money, so can need multiple battles.
+            "Act 3 - Middle Holo Pelt": self.has_inspectometer_battery,  # Can be reached without but possible soft lock
+            "Act 3 - Forest Holo Pelt": self.has_inspectometer_battery,
+            "Act 3 - Crypt Holo Pelt": self.has_inspectometer_battery,
+            "Act 3 - Tower Holo Pelt": self.has_gems_and_battery,
+            "Act 3 - Trader 1": self.has_one_pelt,
+            "Act 3 - Trader 2": self.has_two_pelt,
+            "Act 3 - Trader 3": self.has_three_pelt,
+            "Act 3 - Trader 4": self.has_four_pelt,
+            "Act 3 - Trader 5": self.has_five_pelt,
+            "Act 3 - Goobert's Painting": self.has_gems_and_battery,
+            "Act 3 - The Great Transcendence": self.has_transcendence_requirements,
+            "Act 3 - Boss Mycologists": self.has_mycologists_boss_requirements,
+            "Act 3 - Bone Lord Room": self.has_bone_lord_room_requirements,
+            "Act 3 - Luke's File Entry 1": self.has_battery_and_quill,
+            "Act 3 - Luke's File Entry 2": self.has_battery_and_quill,
+            "Act 3 - Luke's File Entry 3": self.has_battery_and_quill,
+            "Act 3 - Luke's File Entry 4": self.has_transcendence_requirements,
+            "Act 3 - Well": self.has_inspectometer_battery,
+            "Act 3 - Gems Drone": self.has_inspectometer_battery,
+            "Act 3 - Clock": self.has_gems_and_battery,  # Can be brute-forced, but the solution needs those items.
+        }
+        self.region_rules = {
+            "Act 2": self.has_act2_requirements,
+            "Act 3": self.has_act3_requirements,
+            "Epilogue": self.has_epilogue_requirements
+        }
+
+    def has_wardrobe_key(self, state: CollectionState) -> bool:
+        return state.has("Wardrobe Key", self.player)
+
+    def has_caged_wolf(self, state: CollectionState) -> bool:
+        return state.has("Caged Wolf Card", self.player)
+
+    def has_dagger(self, state: CollectionState) -> bool:
+        return state.has("Dagger", self.player)
+
+    def has_magnificus_eye(self, state: CollectionState) -> bool:
+        return state.has("Magnificus Eye", self.player)
+
+    def has_useful_act1_items(self, state: CollectionState) -> bool:
+        return state.has("Oil Painting's Clover Plant", self.player) and state.has("Squirrel Totem Head", self.player)
+
+    def has_all_epitaph_pieces(self, state: CollectionState) -> bool:
+        if self.world.options.epitaph_pieces_randomization.value == 0:
+            return state.has("Epitaph Piece", self.player, 9)
+        elif self.world.options.epitaph_pieces_randomization.value == 1:
+            return state.has("Epitaph Pieces", self.player, 3)
+        else:
+            return state.has("Epitaph Pieces", self.player, 1)
```

Comment:
```suggestion
        if self.world.options.epitaph_pieces_randomization == "all_pieces":
            return state.has("Epitaph Piece", self.player, 9)
        elif self.world.options.epitaph_pieces_randomization == "in_groups":
            return state.has("Epitaph Pieces", self.player, 3)
        else:
            return state.has("Epitaph Pieces", self.player, 1)
```

Would recommend using the string values for the option, or importing the option's class and doing something like
`if self.world.options.epitaph_pieces_randomization == EpitaphPiecesRandomization.option_all_pieces:`